ICAFIX/hcp_fix

```diff
@@ -378,4 +393,16 @@ log_Msg "Done renaming files"
 # Remove the 'fake-NIFTI' file created in fix_3_clean for high-pass filtering of the CIFTI (if it exists)
 $FSLDIR/bin/imrm ${fmrihp}.ica/Atlas
 
+#delete things with too-generic names
+rm -f ${fmrihp}.ica/filtered_func_data.nii.gz ${fmrihp}.ica/Atlas.dtseries.nii
+
+#optional cleanup of hp intermediates
+if [[ "$DeleteIntermediates" == "TRUE" ]]
+then
+    rm -f ${fmrihp}.ica/Atlas_hp_preclean.dtseries.nii ${fmri}_hp${HighPass}.nii.gz
+else
+    #even if we don't delete it, don't leave this file with a hard to interpret name
+    mv -f ${fmrihp}.ica/Atlas_hp_preclean.dtseries.nii ${fmri}_Atlas_hp${HighPass}.dtseries.nii
```

Comment:
Again, variable is `${hp}` and needs to be conditioned on hp>=0.

Reply:
See 9aea92921075857b7f6041ef9d2b6b93ee6c8698.

Reply:
Please feel free to resolve any comments that are no longer an issue, based on either @glasserm's edits in the master branch, or your recent additions.

Reply:
I don't know for certain if Matt's (or my) commits fully resolve the issues you raised.  You identified at least one issue in a later commit - I was thinking you should be the one to mark them as resolved when appropriate, as you raised the issues.

Reply:
Normally, I wouldn't mind doing that, but in this case it is a royal pain.  Is there a way in GitHub to get a consolidated list of the accumulated changes from prior to when the PR was merged to now (the current master branch)?

Reply:
It's not obvious how to do so, but here it is:

https://github.com/Washington-University/HCPpipelines/compare/0f37e01ebec91f711bacf4c04c4abbf55a59af3e...master

What I did to get this is I went to the commit before the PR, clicked "browse files", then clicked "new pull request" (yes, seriously), and then finally, clicked the "change the base" link in the middle of the empty diff.

Reply:
That's handy.  I'll take a look at that, or an updated version thereof, once Matt has had a chance to catch up later.